pkgs/applications/version-management/git-and-tools/git-workspace/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchFromGitHub, rustPlatform, Security }:
+rustPlatform.buildRustPackage rec {
+  pname = "git-workspace";
+  version = "0.4.1";
+  src = fetchFromGitHub {
+    owner = "orf";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "01qxm00c5wqpy1clrvjr44v7cg4nqawaf5a6qnvvgswvis4kakzr";
+  };
+  cargoSha256 = "16rkmk888alfvq8nsggi26vck1c7ya0fa5j7gv219g5py4gw2n34";
+  verifyCargoDeps = true;
+  buildInputs = with stdenv; lib.optional isDarwin Security;
+  meta = with stdenv.lib; {
+    description = "Sync personal and work git repositories from multiple providers";
+    homepage = src.meta.homepage;
+    license = with licenses; [ mit ];
+    platforms = platforms.all;
+    maintainers = with maintainers; [ misuzu ];
+  };
+}
```

Comment:
This expression lacks any proper whitespace, see https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/pantheon/granite/default.nix

Reply:
> This expression lacks any proper whitespace, see https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/pantheon/granite/default.nix

Will do.
Is there some kind of style guide? [nixfmt](https://nixfmt.serokell.io) says that everything is fine as it is now.

Reply:
Unfortunately there isn't. And I think nixfmt is more for formatting nix code and not nixpkgs, and in growing popularity https://github.com/nix-community/nixpkgs-fmt. But I can ask many many of the core body that expressions without whitespace separating attributes is pretty ugly.

I'm not sure if the coding conventions mentions this https://nixos.org/nixpkgs/manual/#chap-conventions, but a cursory look at nixpkgs hopefully reveals this (recent expressions).